src/views/Correction.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <section>
-    <header class="mt-4">
+    <header>
```

Comment:
The parent specifies the whitespace around this component.

Same for child components below (ie, THIS component specifies the whitespace around them).